### PR TITLE
Blog post re-design

### PR DESCRIPTION
**Pull request by KennethWangDotDev**
I made various design changes to make your blog look better. In addition, I have removed the `_aside.scss` (which is now deprecated) and have replaced it with `_article.scss`. 

Full-width content also works better now. You can enable it on an element by using HTML `<p class="full-width">` or Markdown `{:.fullwidth}`

[Here is an album of the Before & After changes](http://imgur.com/a/CS8ge). Every odd image is the _before_. Every even image is the _after_.

### CODE REVIEW

**Comment by fj** [comment]
Comments:
- This looks very nice, but I do want to keep the post icons on the left-hand side. The left side of the text should still line up with the left-bracket of `{ jxf }` at the top, as it does now.
- Does adjusting the title size as you did make the proportions look strange on the blog list page? IIRC they share some styles.

**Comment by KennethWangDotDev** [comment]
- I'll keep the post icon on the left hand side in the next update in a couple hours
- [This](http://i.imgur.com/QFejwCD.png) is what the blog looks like on my latest compile, and the text is lined up with { jxf } at the top. If that's not the case for your version too, then I'll do another `git pull` to fix any versioning issues
- I double checked the blog list page and all font sizes are still the same. I did notice that the blog list items were a little bit off centered. Is this what you meant?
- I will do some tiny design improvements on the blog list page too. Do you also want the metadata to be moved below the title on the blog list page, or is it fine to keep them on the left hand side?

**Comment by fj** [comment]
> This is what the blog looks like on my latest compile, and the text is lined up with { jxf } at the top. If that's not the case for your version too, then I'll do another git pull to fix any versioning issues

This looks good in the screenshots, thanks. I don't see the icon yet, though; did I miss a commit?

> I double checked the blog list page and all font sizes are still the same. I did notice that the blog list items were a little bit off centered. Is this what you meant?

Let's fix that off-centered-ness, but it looks like you're right, the font sizes on the blog list page are good.

> I will do some tiny design improvements on the blog list page too. Do you also want the metadata to be moved below the title on the blog list page, or is it fine to keep them on the left hand side?

No, move it there too -- then they can share the same include, if that makes sense to do.

**Comment by KennethWangDotDev** [comment]
> This looks good in the screenshots, thanks. I don't see the icon yet, though; did I miss a commit?

No, I was just making sure that that we did not have any versioning differences. I will make a commit today

**Comment by KennethWangDotDev** [comment]
Updated

File: _src/blog.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
 layout: page
 ---
 
-<div class="measure--lg">
-  <h1 class="h2 txt--center">Open Brackets</h1>
+<div class="measure--lg blog_post_list">
+  <h1 class="h2 txt--center">Open Bracketsss</h1>
```
Comment (fj) [inline _src/blog.html]: Typo?

**Comment by KennethWangDotDev** [comment]
My bad, fixed the typo.

**Comment by fj** [comment]
Looks good! Also, can you update the README to check off the items you think are done so I can keep track?

**Comment by KennethWangDotDev** [comment]
I just updated the layout for Event entries too.

These are the completed items. If you feel that there is more work for me to do on these items, let me know.
- [X] Add sidenotes (SNs)
- [X] Add margin notes (MNs)
- [X] Add full width content
- [X] Convert existing wide tables to full-width content
- [X] Remove "main content is X% wide" CSS hacks in `_aside.scss` after MNs/SNs transition
- [X] Use YYYY-MM-DD dates in shortlists on homepage; adjust spacing as needed
- [X] Remove dangling left metadata column from lists and individual entries; move metadata below title

---

Remaining tasks:
- [ ] **Convert existing RHS content to MNs / SNs.** _I haven't done this yet because I'm not sure when you would want a sidenote and when you would want a margin._
- [ ] **Full event list should use portfolio-page-like template instead of blog-list-like template.** _Do you mean that you want your All Events page to look similar to your Work (Portfolio) page? I can do that, but the Work (Portfolio) page has an image for each entry, meanwhile there are no images associated with each Event._

**Comment by fj** [comment]
This looks good. I can't remember, did I already clarify everything you need for the last two bullets?

**Comment by KennethWangDotDev** [comment]
No, you haven't yet. Once you do, I'll get right on it :+1:

**Comment by fj** [comment]
> Convert existing RHS content to MNs / SNs. I haven't done this yet because I'm not sure when you would want a sidenote and when you would want a margin.

Make everything a margin note (i.e., it won't have a number).

> Full event list should use portfolio-page-like template instead of blog-list-like template. Do you mean that you want your All Events page to look similar to your Work (Portfolio) page? I can do that, but the Work (Portfolio) page has an image for each entry, meanwhile there are no images associated with each Event.

You got it. For the images, you can just use [placeholder images](https://placehold.it/) and lorem ipsum text.

To make the naming less confusing I'm imagining some name changes:
- `showcase` for the template that includes the showcase items
- `showcase-item` to represent one item in the showcase
- `portfolio.html` for the page that contains the showcase items representing my portfolio
- `events.html` for the page that contains the showcase items representing the events

**Comment by fj** [comment]
This PR is getting big, so I'm going to tidy it up and merge; let's put the last two items on a separate PR.

**Pull request merged**